common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -266,12 +270,27 @@ public Integer run() {
                             logger.debug("{}: {}", file, somaxconn);
                         }
                     } else {
-                        if (logger.isDebugEnabled()) {
-                            logger.debug("{}: {} (non-existent)", file, somaxconn);
+                        // Try to get from sysctl
+                        Integer tmp = null;
+                        if (SystemPropertyUtil.getBoolean("io.netty.net.somaxconn.trySysctl", true)) {
```

Comment:
@normanmaurer - FYI. This is to get the `somaxconn` systems that don't have `/proc` by default (e.g. MacOS). I can set this property to `false` by default if we don't want to start-up new processes during initialization ... not sure if this will be too heavy weight.